server/rpc/handle_get_block_template.go

```diff
@@ -131,6 +129,17 @@ func handleGetBlockTemplate(s *Server, cmd interface{}, closeChan <-chan struct{
 	}
 }
 
+func isSyncedForMining(s *Server) bool {
+	const maxTipAge = 5 * time.Minute
```

Comment:
Please add a comment here or above the function explaining this five minute choice.